website/ref/math.md

```diff
@@ -0,0 +1,16 @@
+<!-- toc -->
+
+# Introduction
+
+The `math:` module provides mathematical functions and
+constants. Everyplace a `$float64` is referenced below that data type
```

Comment:
This explanation is no longer needed. Just add a reference to It now suffices to just refer to https://elv.sh/ref/builtin.html#commands-that-operate-on-numbers at the end of the next paragraph (row 14).

Reply:
Done